packages/@lwc/rollup-plugin/package.json

```diff
@@ -11,10 +11,15 @@
     "bugs": {
         "url": "https://github.com/salesforce/lwc/issues"
     },
-    "main": "src/index.js",
+    "main": "dist/commonjs/index.js",
+    "types": "dist/types/index.d.ts",
```

Comment:
I see. Is there a way to not changing this? Or do we just need to mention this in release note or some documentation?

Reply:
The modern Node way of doing this is to define an [`"exports"`](https://nodejs.org/api/packages.html#exports) field in `package.json`. This avoids allowing consumers to `require`/`import` any file they want.

We don't do this, so I think we should just mention it in the release notes. I wouldn't release a major version bump, though, since I would be surprised if this broke anyone.